preDeploy.sh

```diff
@@ -0,0 +1,42 @@
+#
+# (C) Copyright IBM Corp. 2021, 2021
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -xe
+echo "pre deploy code goes here"
```

Comment:
Nitpick. Boiler plate output is kind of ugly. Something like "Executing preDeploy.sh" would be preferable.

Reply:
Happy to change it.